data.py

```diff
@@ -153,4 +167,5 @@ def skolemise(node):
 
 if __name__ == "__main__":
     oxigraph_url = os.getenv("GRAPH_DB_URL", None)
+    context_file = Path("src/store/metadata/context.json")
```

Comment:
I see what you were doing now, but this script isnt always called by main (if you look at the Makefile you'll see a whacky inline python way of calling). Try it yourself with `make data`.

either way we need setting context in its own function as that local context file is a temporary thing, its (very likely) going to be more complicated than that and I'd look it encapsulated in its own obvious "change me" place.